### DISCUSSION TITLE
Localization support

### DISCUSSION THREAD

**Discussion by kishaningithub** [brianvoe/gofakeit#172] [Ideas]
What are your thoughts on adding localization support for this package?

Got this idea after seeing a faker implementation in python

https://github.com/joke2k/faker

**Comment by kishaningithub** [comment]
Their localization API

https://github.com/joke2k/faker#localization

**Comment by brianvoe** [comment]
I am a dummy that only knows one language. In order for me to add localization I would need people that could properly implement at least another 5 languages that would not only be able to do simple names but addresses and a few other things.
